frontend/e2e-tests/page-objects/input/VotersVotesPgObj.ts

```diff
@@ -2,22 +2,30 @@ import { type Locator, type Page } from "@playwright/test";
 
 import { InputBasePage } from "./InputBasePgObj";
 
-interface VotersCounts {
-  poll_card_count: string;
-  proxy_certificate_count: string;
-  voter_card_count: string;
-  total_admitted_voters_count: string;
+export interface VotersCounts {
+  poll_card_count: number;
+  proxy_certificate_count: number;
+  voter_card_count: number;
+  total_admitted_voters_count: number;
 }
 
-interface VotesCounts {
-  votes_candidates_counts: string;
-  blank_votes_count: string;
-  invalid_votes_count: string;
-  total_votes_cast_count: string;
+export interface VotesCounts {
+  votes_candidates_counts: number;
+  blank_votes_count: number;
+  invalid_votes_count: number;
+  total_votes_cast_count: number;
+}
+
+export interface VotersRecounts {
+  poll_card_recount: number;
+  proxy_certificate_recount: number;
+  voter_card_recount: number;
+  total_admitted_voters_recount: number;
 }
```

Comment:
Can't you import these from `openapi.ts`?

Reply:
I don't think we should. If at some point the API specs change (however unlikely for these particular fields), that might then break the test code.